### PR TITLE
GPL copyright statements

### PR DESCRIPTION
**Pull request by dgreenhalgh**
in order to be protected under the GPL, we need to add a copyright statement to the top of every source file in our project. 
I've added three files to docs. 
copyright.txt is our general copyright statement, 
copyrightJava.txt is a version of the copyright statement formatted into a Java-style comment,
and copyrightXML.txt is a version of the copyright statement formatted into an XML-style comment.

I plan on adding these to the top of every existing file using sed, but it's up to you guys to add a copyright statement to the top of all files created after these are put into place.

Let me know if there are any changes that you'd like me to make before they're added to the top of every file because it could be a pain to update them.

### CODE REVIEW

File: docs/copyrightJava.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (ejohnson44) [inline docs/copyrightJava.txt:1]: Do we want this as a javadoc? Or just as a regular comment?

Comment (dgreenhalgh) [inline docs/copyrightJava.txt:1]: Good call

**Comment by dgreenhalgh** [comment]
Motherfucker. Alright so I only meant to add a change to that one line and I accidentally pushed a bunch of commits intended to cleanup (delete) our unused import statements.

So now there's a bit more code to review.

File: SoundStream/src/com/lastcrusade/soundstream/components/ConnectFragment.java [hunk 2]
```diff
@@ -39,10 +39,14 @@
     private BroadcastRegistrar broadcastRegistrar;
     private Button connectButton;
     
+    private ServiceLocator<MusicLibraryService> musicLibraryLocator;
```
Comment (ejohnson44) [inline SoundStream/src/com/lastcrusade/soundstream/components/ConnectFragment.java:42]: I don't think this stuff is supposed to be here anymore - i think that was what @theJenix removed earlier today

Comment (theJenix) [inline SoundStream/src/com/lastcrusade/soundstream/components/ConnectFragment.java:42]: @ejohnson44 is correct..this was moved to CustomApp.  See the the "fan to host library" pull request.

**Comment by reidbaker** [comment]
Feature branches :wink:

**Comment by dgreenhalgh** [comment]
I used one, just messed it up

**Comment by reidbaker** [comment]
Sorry my bad then. :thumbsup:

**Comment by dgreenhalgh** [comment]
See pull request #73

**Pull request closed without merging**